plugins/search-backend-module-techdocs/config.d.ts

```diff
@@ -19,10 +19,33 @@ import { TaskScheduleDefinitionConfig } from '@backstage/backend-tasks';
 export interface Config {
   search?: {
     collators?: {
+      /**
+       * Configuration options for `@backstage/plugin-search-backend-module-techdocs`
+       */
       techdocs?: {
+        /**
+         * The schedule for how often to run the collation job.
+         */
         schedule?: TaskScheduleDefinitionConfig;
+        /**
+         * A templating string with placeholders, to form the final location of
+         * the entity.
+         *
+         * @defaultValue '/docs/:namespace/:kind/:name/:path'
```

Comment:
Dunno about this tbh, it doesn't show up properly in the config docs. It's actually broken for our API reference too, these tags don't show up. In editors you'll get the literal tag before, so not particularly easier to read either imo.

Prolly best to stick to just mentioning in text for now.